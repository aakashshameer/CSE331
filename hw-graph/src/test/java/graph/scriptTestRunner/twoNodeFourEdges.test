#Test to create two nodes, two self-edges and two edges between different nodes

#Create a graph
CreateGraph graphName

#Add two nodes
Added graphName b
Added graphName c

#Add two self edges
AddEdge graphName b b BB
AddEdge graphName c c CC

#Add two edges between different nodes
AddEdge graphName b c BC
AddEdge graphName c b CB

#Print the nodes in the graph
ListNodes graphName

#Print outgoing edges from b
ListChildren graphName b

#Print outgoing edges from c
ListChildren graphName c