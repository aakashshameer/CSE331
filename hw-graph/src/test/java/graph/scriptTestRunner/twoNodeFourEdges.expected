#Test to create two nodes, two self-edges and two edges between different nodes

#Create a graph
create graph graphName

#Add two nodes
added node b to graphName
added node c to graphName

#Add two self edges
added edge BB from b to b in graphName
added edge CC from c to c in graphName

#Add two edges between different nodes
added edge BC from b to c in graphName
added edge CB from c to b in graphName

#Print the nodes in the graph
graphName contains: b c

#Print outgoing edges from b
the children of b in graphName are: b(BB) c(BC)

#Print outgoing edges from c
the children of c in graphName are: b(CB) c(CC)